frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -34,44 +34,52 @@ export const PropertyKeyInfo = React.forwardRef<HTMLSpanElement, PropertyKeyInfo
 
     value = value?.toString() ?? '' // convert to string
 
-    const data = getCoreFilterDefinition(value, type)
-    const valueDisplayText = (data ? data.label : value)?.trim() ?? ''
+    const coreDefinition = getCoreFilterDefinition(value, type)
+    const valueDisplayText = (coreDefinition ? coreDefinition.label : value)?.trim() ?? ''
     const valueDisplayElement = valueDisplayText === '' ? <i>(empty string)</i> : valueDisplayText
 
+    const recognizedSource: 'posthog' | 'langfuse' | null = coreDefinition
+        ? 'posthog'
+        : value.startsWith('langfuse ')
```

Comment:
I made the assumption that all events coming from the official Langfuse integration _start_ with "langfuse ". This is so that we recognize _only_ those events as official. Let me know if this assumption may be wrong